changelogs/8.15.asciidoc

```diff
@@ -1,10 +1,22 @@
 [[apm-release-notes-8.15]]
 == APM version 8.15
 
+* <<apm-release-notes-8.15.3>>
 * <<apm-release-notes-8.15.2>>
 * <<apm-release-notes-8.15.1>>
 * <<apm-release-notes-8.15.0>>
 
+[float]
+[[apm-release-notes-8.15.3]]
+=== APM version 8.15.3
+
+https://github.com/elastic/apm-server/compare/v8.15.2\...v8.15.3[View commits]
+
+[float]
+==== Bug fixes
+
+- Fix a panic in OTLP intake protocol, occurring when invalid format of data was included in arrays {pull}13703[13703]
+
 [float]
 [[apm-release-notes-8.15.2]]
 === APM version 8.15.2
```

Comment:
Can you delete the relevant lines (30 and 35) in 8.15.2, they are moved to 8.15.3. I hate the github review UI.